.github/workflows/unit-tests.yml

```diff
@@ -234,7 +234,8 @@ jobs:
           /opt/python/cp311-cp311/bin/python -m venv ~/virtualenvs/pandas-dev
           . ~/virtualenvs/pandas-dev/bin/activate
           python -m pip install --no-cache-dir -U pip wheel setuptools meson[ninja]==1.2.1 meson-python==0.13.1
-          python -m pip install --no-cache-dir versioneer[toml] cython numpy python-dateutil pytz pytest>=7.3.2 pytest-xdist>=2.2.0 pytest-asyncio>=0.17 hypothesis>=6.46.1
+          # No longer possible to easily compile numpy > 1.26 from source
```

Comment:
What makes the source install difficult?

Reply:
Curious about this too. Are the tests not even able to run due to installing numpy?

Reply:
numpy needs a BLAS which is not present on Github Actions to build, by default.

I didn't feel like spending time to fiddle around with the config settings for 32-bit (given its probably going to be removed sometime soon-ish), so I just left it. IMO, it's not the end of the world that its pinned, but if you want me to try to fiddle around, I can set some time towards the end of this week.

Reply:
Are you referring to this failure? https://github.com/pandas-dev/pandas/commit/ebca6dfb8f159ddc00081a463a6d9eda0fc96d8c

Reply:
Ah, nice.

Thanks for the fix!